samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', filepath = 'path/to/image.png') {
+  // [START dlp_inspect_image_file]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const filepath = 'path/to/image.png';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
```

Comment:
Why is this uppercase?

Why not:

```js
const dlp = require('@google-cloud/dlp');

// ...

const client = new dlp.DlpServiceClient();
```

Reply:
If `DLP` is your preference, can it be `Dlp`? Or does full capitalization need to be configurable?

Reply:
Done